modules/Rock.cmake

```diff
@@ -1350,6 +1366,39 @@ function(rock_gtest TARGET_NAME)
 
     rock_setup_gtest_test(${TARGET_NAME} ${GMOCK_DIR} ${GTEST_DIR})
     rock_add_test(${TARGET_NAME} "${__rock_test_parameters}")
+    list(APPEND ALL_TEST_TARGETS "test-${TARGET_NAME}-cxx")
+    set(ALL_TEST_TARGETS "${ALL_TEST_TARGETS}" CACHE INTERNAL "ALL_TEST_TARGETS")
+endfunction()
+
+function(rock_cxx_coverage_report PROJECT_NAME)
+    message(STATUS "Generating report for ${PROJECT_NAME} package")
+    find_program(
+        gcovr_exec NAMES
+        ${ROCK_GCOVR_EXECUTABLE} gcovr
+    )
+    if (NOT gcovr_exec)
+        message(FATAL_ERROR "Could not find an executable for gcovr.")
+    endif()
+
+    if(${ROCK_COVERAGE_REPORT_PATH})
+        message(STATUS "Setting an explicit config path coverage report")
```

Comment:
When this happens, it's good to actually show the path (i.e. "Coverage report will be generated in ${ROCK_COVERAGE_REPORT_PATH}")